.github/workflows/nix-update.yml

```
@@ -0,0 +1,11 @@
+name: Nix update
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+
+jobs:
+  nix-update:
+    uses: enarx/.github/.github/workflows/shared-nix-update.yml@main
```

```suggestion
    uses: enarx/.github/.github/workflows/nix-update.yml@main
```

Are we going to have any non-shared workflows in https://github.com/enarx/.github?